src/main/webapp/script.js

```diff
@@ -555,7 +534,7 @@ function createQuestionElement(question, isForum) {
   // Star icon.
   const iconElement = document.createElement('i');
   iconElement.setAttribute('id', 'icon' + question.id);
-  iconElement.setAttribute('style', 'cursor: pointer');
+  iconElement.setAttribute('style', 'cursor: pointer; display: none');
```

Comment:
Perhaps a more descriptive name like starIconElement would make it easier to understand. You do have the comment which makes this just a suggestion and not a request.

Reply:
done